tests/xharness/Jenkins/TestSelector.cs

```diff
@@ -201,68 +208,67 @@ void SelectTestsByLabel (int pullRequest)
 			}
 
 			MainLog.WriteLine ($"In total found {labels.Count ()} label(s): {string.Join (", ", labels.ToArray ())}");
+			
+			// first check if the all label is used, we either skip our run all, then check the skips, then run,
```

Comment:
Not sure why the previous logic in `SetEnabled` to establish this precedence doesn't work anymore? Not changing existing logic seems like the better option as opposed to writing new logic.

Reply:
I'll revert to that with flags, will leave the tests. To much work for no gain.